microsetta_interface/templates/sample.jinja2

```diff
@@ -31,10 +31,28 @@
             return false;
         }
 
+        function setSampleMetaVisibility(sample_site) {
+            let display_state = "";
+
+            // If the sample site is not Cheek, we will hide all of the divs and set the field values to ""
+            if (sample_site != "Cheek") {
+                display_state = "none";
+                $("#sample_site_last_washed_date").val("");
+                $("#sample_site_last_washed_time").val("");
+                $("#sample_site_last_washed_product").val("").change();
+            }
```

Comment:
I'm a little surprised that `sample_site_last_washed_date_normalized` isn't also blanked out at the same time, for the potential case where the user initially sets something as a cheek sample but then changes it.  Or is that case handled at submission by the `monthDayYearCheekWash` validation?

Reply:
Good catch, thanks.